content/docs/concepts/architecture/index.md

```diff
@@ -38,10 +38,6 @@ The *api* component provides a REST API that allows communicating with Keptn. It
 
```

Comment:
Regarding the *api-service*: What is actually its job now because we don`t have an eventbroker anymore. 
The sentences:
*  ~~However, the api component mostly uses other internal services or forwards events to the *eventbroker*~~ 
* ~~In addition to the REST API, the api component maintains a websocket server to forward Keptn messages to the *Keptn CLI*~~
are obsolete.

Reply:
I have updated the description accordingly